[2025-02-27T21:39:35.319+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-27T21:39:35.334+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_and_transform.create_table_usa manual__2025-02-27T21:34:00.844211+00:00 [queued]>
[2025-02-27T21:39:35.342+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_and_transform.create_table_usa manual__2025-02-27T21:34:00.844211+00:00 [queued]>
[2025-02-27T21:39:35.343+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-02-27T21:39:35.358+0000] {taskinstance.py:2890} INFO - Executing <Task(BigQueryInsertJobOperator): create_table_usa> on 2025-02-27 21:34:00.844211+00:00
[2025-02-27T21:39:35.365+0000] {standard_task_runner.py:72} INFO - Started process 6005 to run task
[2025-02-27T21:39:35.372+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'load_and_transform', 'create_table_usa', 'manual__2025-02-27T21:34:00.844211+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/transformation.py', '--cfg-path', '/tmp/tmphpaz6wai']
[2025-02-27T21:39:35.376+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask create_table_usa
[2025-02-27T21:39:35.432+0000] {task_command.py:467} INFO - Running <TaskInstance: load_and_transform.create_table_usa manual__2025-02-27T21:34:00.844211+00:00 [running]> on host airflow.us-central1-c.c.healthcare-project-452218.internal
[2025-02-27T21:39:35.504+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='load_and_transform' AIRFLOW_CTX_TASK_ID='create_table_usa' AIRFLOW_CTX_EXECUTION_DATE='2025-02-27T21:34:00.844211+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-27T21:34:00.844211+00:00'
[2025-02-27T21:39:35.506+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-27T21:39:35.536+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-02-27T21:39:35.537+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-02-27T21:39:35.538+0000] {credentials_provider.py:409} INFO - Getting connection using `google.auth.default()` since no explicit credentials are provided.
[2025-02-27T21:39:35.558+0000] {bigquery.py:2956} INFO - Executing: {'query': {'query': "\n                        CREATE OR REPLACE TABLE `healthcare-project-452218.transform_dataset.usa_table` AS\n                        SELECT *\n                        FROM `healthcare-project-452218.staging_dataset.global_data`\n                        WHERE country = 'USA'\n                    ", 'useLegacySql': False}}'
[2025-02-27T21:39:35.568+0000] {bigquery.py:1344} INFO - Inserting job airflow_load_and_transform_create_table_usa_2025_02_27T21_34_00_844211_00_00_fb0517167e9da8d59d00846e944073c0
[2025-02-27T21:39:39.112+0000] {bigquery.py:2927} INFO - Job airflow_load_and_transform_create_table_usa_2025_02_27T21_34_00_844211_00_00_fb0517167e9da8d59d00846e944073c0 is completed. Checking the job status
[2025-02-27T21:39:39.129+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-27T21:39:39.130+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=load_and_transform, task_id=create_table_usa, run_id=manual__2025-02-27T21:34:00.844211+00:00, execution_date=20250227T213400, start_date=20250227T213935, end_date=20250227T213939
[2025-02-27T21:39:39.203+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-27T21:39:39.251+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-27T21:39:39.253+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
